scripts/dev-server.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-env node */
```

Comment:
I played with the rollup plugin api tonight and we may be able to do this without this script.  Not ready to PR to this, so I'll just show it to you tomorrow and we can discuss.

This works fine by the way of course, but, since we're relying on the rollup serve plugin anyway, keeping it in rollup would make sense.